src/main/fc/fc_msp.c

```diff
@@ -399,6 +399,13 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
         }
         break;
 
+    case MSP2_INAV_VOLTAGE_METER_CONFIG:
```

Comment:
Do we need a special MSPv2 message for voltmeter config? Same settings are available via `MSP2_INAV_BATTERY_CONFIG`.

Reply:
Indeed we do not. I added it with the PR increasing the voltage resolution but now it is redundant I will remove it.